### ISSUE TITLE
Pop-up not shown when used in a DrawerLayout/NavigationView

### ISSUE THREAD

**Issue by HughJeffner** [jaredrummler/MaterialSpinner#128]
I am using the spinner in the header layout of a NavigationView used in a DrawerLayout

When the app starts up everything works normally, but after selecting a navigation item in the NavigationView and re-opening the drawer the pop-up no longer shows when clicking on the spinner.

I traced back the problem to this line: 

https://github.com/jaredrummler/MaterialSpinner/blob/3dc38b50ab9a64971801e350b40a23a5c8fb98d1/library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinner.java#L524

For some reason `isLaidOut()` returns false at that point so`canShowPopup` returns false.  Not sure if this is some kind of interaction with the support libraries (I am using androidX) when running on newer API devices.  I tested this on a few API levels and the problem seems to be happening on Nougat (API 24)+

As a work-around I added a listener to the drawer opened event and called `requestLayout()` on the spinner.

**Comment by AshutoshChakraborty** [comment]
Yes In my case also, I am using Spinner inside a RecyclerView as a child, the specified issue occurred.
I have also use the solution suggested @HughJeffner and use `requestLayout()`  in My ViewHolder when i initialise the Spinner, and it works fine.

**Comment by eygraber** [comment]
Same problem here. Kinda gross to have to `requestLayout` in the ViewHolder, especially since the width and height are set correctly.
